docs/en/stack/security/authentication/realms.asciidoc

```diff
@@ -2,9 +2,12 @@
 [[realms]]
 === Realms
 
-Authentication in {security} is handled by one or more authentication services
-called _realms_. A _realm_ is used to resolve and authenticate users based on
-authentication tokens. {security} provides the following built-in realms:
+The {stack-security-features} authenticate users by using one or more
+<<token-authentication-services,token-based authentication services>> and realms
+defined in a _realm chain_.
```

Comment:
```suggestion

```

Reply:
I've suggested removing this, since I don't think it's necessary to go into the concept of chains in this page and it made the relationship of chains to token services unclear.